tests/system/runtimes/test_databricks.py

```diff
@@ -155,20 +161,69 @@ def test_kwargs_from_file(self):
             image="mlrun/mlrun",
         )
 
-        self._add_databricks_env(function=function, is_cluster_id_required=True)
+        self._add_databricks_env(function=function)
 
         run = function.run(
             handler="func",
             auto_build=True,
             project="databricks-proj",
-            params={
-                "task_parameters": {"timeout_minutes": 15},
-                "param1": "value1",
-                "param2": "value2",
-            },
+            params=default_test_params,
         )
         assert run.status.state == "completed"
         assert (
             run.status.results["databricks_runtime_task"]["logs"]
             == "{'param1': 'value1', 'param2': 'value2'}\n"
         )
+
+    @pytest.mark.parametrize(
+        "handler, function_name",
+        [
+            ("print_kwargs", "print_kwargs"),
+            (
+                "",
+                "handler",
+            ),  # test default handler.
+        ],
+    )
+    def test_rerun(self, handler, function_name):
+        databricks_code = print_kwargs_function % function_name
+        function_kwargs = {"handler": handler} if handler else {}
+        function_ref = FunctionReference(
+            kind="databricks",
+            code=databricks_code,
+            image="mlrun/mlrun",
+            name="databricks-test",
+        )
+
+        function = function_ref.to_function()
+
+        self._add_databricks_env(function=function)
+        run = function.run(
+            project="databricks-proj", params=default_test_params, **function_kwargs
+        )
+        self.assert_print_kwargs(print_kwargs_run=run)
+        second_run = function.run(runspec=run, project="databricks-proj")
+        self.assert_print_kwargs(print_kwargs_run=second_run)
+
+    def test_missing_code_run(self):
+        function_ref = FunctionReference(
+            kind="databricks",
+            code="",
+            image="mlrun/mlrun",
+            name="databricks-test",
+        )
+
+        function = function_ref.to_function()
+
+        self._add_databricks_env(function=function)
+        with pytest.raises(mlrun.errors.MLRunBadRequestError) as bad_request_error:
+            run = function.run(
+                project="databricks-proj",
+                params=default_test_params,
+                handler="not_exist_handler",
+            )
+            assert run.status.state == "error"
+            assert (
+                "Databricks function must be provided with user code"
```

Comment:
I guess you might as well just raise it in `get_internal_parameters` and remove the if-else?